# CanData Message

std_msgs/Header header

std_msgs/UInt32 address
std_msgs/UInt16 busTime
std_msgs/String dat
std_msgs/UInt8 src



# struct CanData {
#   address @0 :UInt32;
#   busTime @1 :UInt16;
#   dat     @2 :Data;
#   src     @3 :UInt8;
# }
